DESCRIPTION

```diff
@@ -29,6 +29,7 @@ BugReports: https://github.com/r-lib/vctrs/issues
 Depends: 
     R (>= 3.3)
 Imports:
+    cli (>= 3.1.1.9000),
```

Comment:
I feel like this is worth a new bullet, saying something like "vctrs now imports cli to help with generation of useful error messages."

Normally I wouldn't mention something like this, but since vctrs is nearly "free", this seems worth mentioning